From: James Tocknell <aragilar@gmail.com>
Date: Wed, 14 Oct 2015 01:02:58 +1100
Subject: Fix library paths for multiarch

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

Index: sundials-6.1.1/CMakeLists.txt
===================================================================
--- sundials-6.1.1.orig/CMakeLists.txt
+++ sundials-6.1.1/CMakeLists.txt
@@ -39,6 +39,10 @@ include(CheckCSourceCompiles)
 include(FindPackageHandleStandardArgs)
 
 
+# Get correct build paths automatically
+
+INCLUDE(GNUInstallDirs)
+
 # Set some variables with info on the SUNDIALS project
 set(PACKAGE_BUGREPORT "sundials-users@llnl.gov")
 set(PACKAGE_NAME "SUNDIALS")
