Description: Add some more headers into the installation
Author: Anton Gladky <gladk@debian.org>
Last-Update: 2021-03-27

Index: sundials-6.4.1+dfsg1/CMakeLists.txt
===================================================================
--- sundials-6.4.1+dfsg1.orig/CMakeLists.txt
+++ sundials-6.4.1+dfsg1/CMakeLists.txt
@@ -280,3 +280,13 @@ export(
   FILE "${CMAKE_CURRENT_BINARY_DIR}/SUNDIALSTargets.cmake"
   NAMESPACE SUNDIALS::
 )
+install(FILES "src/ida/ida_impl.h" "src/ida/ida_ls_impl.h"
+        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/ida"
+)
+install(FILES "src/kinsol/kinsol_impl.h" "src/kinsol/kinsol_ls_impl.h"
+        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/kinsol"
+)
+install(FILES "src/sundials/sundials_context_impl.h" "src/sundials/sundials_logger_impl.h" "src/sundials/sundials_hashmap.h"
+        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/"
+)
+
